components/script/dom/htmlareaelement.rs

```diff
@@ -240,13 +241,13 @@ impl HTMLAreaElement {
     pub fn get_shape_from_coords(&self) -> Option<Area> {
        let elem = self.upcast::<Element>();
        let shape = elem.get_string_attribute(&"shape".into());
-       let shp: Shape = match shape.to_lowercase().as_ref() {
-           "circle" => Shape::Circle,
-           "circ" => Shape::Circle,
-           "rectangle" => Shape::Rectangle,
-           "rect" => Shape::Rectangle,
-           "polygon" => Shape::Rectangle,
-           "poly" => Shape::Polygon,
+       let shp: Shape = match &shape {
```

Comment:
Addressed in https://github.com/servo/servo/pull/17883/commits/719b09ca60920e91289b75550bfd0e572246d01d